### PR TITLE
wire up BlockIndexPage

### LINKED ISSUE #204: Block index page: wire up with load more

### PR DESCRIPTION
**Pull request by prin-r**
Fixes #204

### CODE REVIEW

File: scan/src/components/TxsTable.re [hunk 1]
```diff
@@ -1,109 +1,56 @@
...
+        <Col size=0.6>
           <div className=TElement.Styles.feeContainer>
             <Text block=true value="FEE" size=Text.Sm weight=Text.Bold color=Colors.grayText />
           </div>
         </Col>
       </Row>
     </THead>
     {txs
-     ->Belt.List.mapWithIndex((idx, {blockHeight, hash, timestamp, gasUsed, messages}) => {
+     ->Belt.List.mapWithIndex((idx, {blockHeight, hash, timestamp, gasUsed, messages, sender}) => {
```
Comment (evilpeach) [inline scan/src/components/TxsTable.re]: Can we use `blockHeight` instead of idx?

File: scan/src/components/TxsTable.re [hunk 2]
```diff
@@ -1,109 +1,56 @@
...
-          codeHash: "0x912381238128381230" |> Hash.fromHex,
-          params: "0x8238233288238238" |> JsBuffer.fromHex,
-          reportPeriod: 23,
-          sender: "0x99329329239239923923" |> Address.fromHex,
-        }),
-      ],
-      events: [],
-    },
-  ];
+let make = (~txs: list(TxHook.Tx.t)=[]) => {
```
Comment (evilpeach) [inline scan/src/components/TxsTable.re]: remove default value

File: scan/src/pages/BlockIndexPage.re [hunk 3]
```diff
@@ -55,37 +59,63 @@ let make = (~height) => {
...
+           <TxsTable txs />
+           <VSpacing size=Spacing.lg />
+           {txs->Belt_List.size < limit
+              ? React.null : <LoadMore onClick={_ => setLimit(oldLimit => oldLimit + 10)} />}
+           <VSpacing size=Spacing.xl />
+         </>
+       }
+     | _ => <VSpacing size={`px(280)} />
+     }}
   </div>;
```
Comment (evilpeach) [inline scan/src/pages/BlockIndexPage.re:120]: @sorawit What's your opinion?

**Review by evilpeach — CHANGES_REQUESTED** [review]

**Review by evilpeach — APPROVED** [review]

**Review by sorawit — APPROVED** [review]
Looks great.

**Pull request merged**